sdk/eventhubs/pom.xml

```diff
@@ -7,11 +7,68 @@
   <artifactId>azure-eventhubs-service</artifactId>
   <packaging>pom</packaging>
   <version>1.0.0</version>  <!-- Need not change for every release-->
-  <modules>
-    <module>microsoft-azure-eventhubs</module>
-    <module>microsoft-azure-eventhubs-eph</module>
-    <module>microsoft-azure-eventhubs-extensions</module>
-    <module>azure-messaging-eventhubs</module>
-    <module>azure-messaging-eventhubs-checkpointstore-blob</module>
-  </modules>
+
+  <profiles>
+    <!-- DO NOT CHANGE THE *ORDER* IN WHICH THESE PROFILES ARE DEFINED! -->
+    <profile>
+      <id>coverage</id>
+      <modules>
+        <module>azure-messaging-eventhubs</module>
+        <module>azure-messaging-eventhubs-checkpointstore-blob</module>
+      </modules>
+
+      <dependencies>
+        <dependency>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-messaging-eventhubs</artifactId>
+          <version>5.4.0-beta.2</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs;current} -->
+        </dependency>
+        <dependency>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
+          <version>1.4.0-beta.2</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;current} -->
+        </dependency>
+      </dependencies>
+
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>0.8.5</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
+            <executions>
+              <execution>
+                <id>report-aggregate</id>
+                <phase>verify</phase>
+                <goals>
+                  <goal>report-aggregate</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>${project.reporting.outputDirectory}/test-coverage</outputDirectory>
+                  <excludes>
+                    <exclude>**/com/azure/cosmos/implementation/apachecommons/**/*</exclude>
+                    <exclude>**/com/azure/cosmos/implementation/guava25/**/*</exclude>
+                    <exclude>**/com/azure/cosmos/implementation/guava27/**/*</exclude>
+                  </excludes>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
+        <module>microsoft-azure-eventhubs</module>
+        <module>microsoft-azure-eventhubs-eph</module>
+        <module>microsoft-azure-eventhubs-extensions</module>
```

Comment:
Should we think of having a `pom.data.xml`, in a similar vain as we have `pom.mgmt.xml` to separate out Track 1 libraries from Track 2?

Reply:
That's another question for EngSys.

Reply:
This may be out of scope for this PR.